test/server/admin/stats_handler_speed_test.cc

```diff
@@ -110,7 +99,6 @@ BENCHMARK(BM_UsedCountersText)->Unit(benchmark::kMillisecond);
 // NOLINTNEXTLINE(readability-identifier-naming)
 static void BM_FilteredCountersText(benchmark::State& state) {
```

Comment:
This benchmark is the same as BM_Re2FilteredCountersText ? Here are elsewhere in the file, delete the BM if there's a RE2 version of it

Reply:
Done